pdns/opensslsigners.cc

```diff
@@ -225,10 +224,10 @@ class OpenSSLRSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
   bool verify(const std::string& hash, const std::string& signature) const override;
   std::string getPubKeyHash() const override;
   std::string getPublicKeyString() const override;
-  std::unique_ptr<BIGNUM, void(*)(BIGNUM*)>parse(std::map<std::string, std::string>& stormap, const std::string& key) const;
+  std::unique_ptr<BIGNUM, void (*)(BIGNUM*)> parse(std::map<std::string, std::string>& stormap, const std::string& key) const;
   void fromISCMap(DNSKEYRecordContent& drc, std::map<std::string, std::string>& stormap) override;
   void fromPublicKeyString(const std::string& content) override;
-  bool checkKey(vector<string> *errorMessages) const override;
+  bool checkKey(vector<string>* errorMessages) const override;
```

Comment:
would be maybe opportune moment to not use vector<T>* here?

Reply:
Agreed, `vector<string>&` is probably better here.

Reply:
I'll add the change to an upcoming PR.